app/config/locale/translations/mr.json

```
@@ -3,31 +3,31 @@
     "settings.locale": "mr",
     "settings.direction": "ltr",
     "emails.sender": "%s टीम",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
-    "locale.country.unknown": "अज्ञात",
+    "emails.verification.subject": "खाते पडताळणी.",
+    "emails.verification.hello": "अहो {{नाव}}.",
+    "emails.verification.body": "आपला ईमेल पत्ता सत्यापित करण्यासाठी या दुव्याचे अनुसरण करा.",
+    "emails.verification.footer": "आपण हा पत्ता सत्यापित करण्यास न विचारल्यास आपण या संदेशाकडे दुर्लक्ष करू शकता.",
+    "emails.verification.thanks": "धन्यवाद.",
+    "emails.verification.signature": "{{प्रकल्प}} संघ.",
+    "emails.magicSession.subject": "लॉगिन करा.",
+    "emails.magicSession.hello": "नमस्कार.",
```

```suggestion
    "emails.magicSession.hello": "नमस्कार,",
```
Please use the correct punctuation in all the strings. You can use `es.json` for reference.

---

pls review https://github.com/omnaladkar/appwrite/commit/190c5dfed1bb09d412df10848406306e45958461